.github/workflows/commit.yaml

```diff
@@ -62,3 +62,25 @@ jobs:
           key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
 
       - run: make test
+
+  bench:
+    name: Benchmark
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+
+      - uses: actions/checkout@v2
+
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cache/go-build
+            ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+
+      - run: make bench
+
+      # TODO: publish results as PR comments if becomes slow?
```

Comment:
good TODO, though not sure how consistent CI perf is to action that!

Reply:
yeah agreed :D remove the comment for now :D